examples/asa/assets/5-contract-owned-asa.py

```diff
@@ -0,0 +1,62 @@
+# Escrow Account Contract
```

Comment:
Can we rename/add a suffix to file name `-stateless` as well? Eg. *5-contract-asa-stateless.py*